test/Test/Network/GRPC/MQTT/Message/Packet.hs

```diff
@@ -57,10 +44,7 @@ tests =
   testGroup
     "Network.GRPC.MQTT.Message.Packet"
     [ testTreePacketWire
-    , testTreePacketSplit
-    , testTreePacketOrder
```

Comment:
Does this mean we no longer have a test for handing out-of-order packets? That feels like an important property we should have a test for.